src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -525,11 +722,16 @@ class ContentExplorer extends Component {
                     itemButtonRenderer={itemButtonRenderer}
                     itemIconRenderer={itemIconRenderer}
                     itemNameLinkRenderer={itemNameLinkRenderer}
-                    items={items}
+                    items={includeSubfolders && !itemsAreSelectedAndActionDisabled ? this.getFoldersOnly() : items}
```

Comment:
some of this looks very specific to how enterprise-reports will use the toggle but it might not be the same for all use cases. could we make it more general and utilize the item renderers in the parent app?